dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/RumFeatureTest.kt

```diff
@@ -386,4 +391,48 @@ internal class RumFeatureTest {
 
         verify(monitor).stopKeepAliveCallback()
     }
+
+    @Test
+    fun `it will register and unregister the provided plugin when initialise and stop called`(
```

Comment:
Just to be safe, I'd split this in 2 tests : verifying that initialize only calls register, and stop only calls unregsiter.